spec/lib/dfid/patch/specialist_publisher/countries_spec.rb

```diff
@@ -5,15 +5,9 @@
 describe DfidTransition::Patch::SpecialistPublisher::Countries do
   subject(:patch) { described_class.new(patch_location) }
 
-  describe '#location' do
-    context 'a schema location is supplied' do
-      let(:patch_location) { 'spec/fixtures/patchme.json' }
-
-      it 'patches that location' do
-        expect(patch.location).to eq(patch_location)
-      end
-    end
+  it_behaves_like "a patcher"
```

Comment:
Yup you're right, it doesn't reveal enough about what being a patcher constitutes, will change.